packages/engine/Specs/Scene/Model/MetadataPipelineStageSpec.js

```diff
@@ -340,6 +342,65 @@ describe(
       });
     });
 
+    it("Adds property texture transform to the shader", function () {
```

Comment:
same here, use `async/await`

Reply:
This is an "auto-generated" file (i.e. written by VSCode glTF plugin in this exact form), so one could make a case to _not_ apply hand-edits. But it will eventually be maintained manually (and human-readable), so that's probably not a huge deal here.